### PR TITLE
Update generated protobuf files

### PR DESCRIPTION
**Pull request by XianzheMa**
We have a big version change of `protobuf` from a previous PR https://github.com/eth-easl/modyn/pull/412. This results in many changes in the generated files. This PR updates them for consistency.

### CODE REVIEW

**Review by robinholzi — COMMENTED** [review]
Changes seem reasonable👍 . Fixing the failing CI will probably be rather tedious...

File: modyn/protos/README.md [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 This assumes python 3.6+ is installed.
 
-## Install the grpcio-tools package
-
-`pip install grpcio-tools`
-
 ## Generate the python files
```
Comment (XianzheMa) [inline modyn/protos/README.md:5]: we already installed `grpcio-tools` in `dev-requirements.txt`. And if the developer follows the README.md, she/he already did `pip install -r dev-requirements.txt` as README.md says.

**Review by robinholzi — APPROVED** [review]
LGTM :)

File: environment.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ dependencies:
   - tqdm
   - conda-forge::enlighten
   - pip:
-    - grpcio==1.59.0
```
[the remark this answers is not in the document]
Comment (MaxiBoether) [reply environment.yml:22]: shouldn't it be >= 1.63?

**Review by MaxiBoether — APPROVED** [review]
Thanks for updating. Just wondering whether we should put >= 1.63 (we pinned 1.59 because newer versions were crashing at some point). But we can also keep that.

**Comment by XianzheMa** [comment]
> Thanks for updating. Just wondering whether we should put >= 1.63 (we pinned 1.59 because newer versions were crashing at some point). But we can also keep that.

Thanks! I changed it to >= 1.63

**Comment by XianzheMa** [comment]
> Thanks for updating. Just wondering whether we should put >= 1.63 (we pinned 1.59 because newer versions were crashing at some point). But we can also keep that.

thanks I changed it

**Pull request merged**